.github/workflows/testing-unit.yaml

```diff
@@ -28,8 +28,11 @@ jobs:
         with:
           include_grpc: '${{ matrix.use_grpc }}'
           include_types: true
-      - name: Run unit tests
+      - name: Run unit tests (REST)
```

Comment:
I think these do cover the REST surface area, but also a lot of the "other" stuff like common utils, etc.

Reply:
Yeah. They're really meant to be "everything that doesn't require GRPC dependencies" unit tests. But that doesn't roll off the tongue 🤣